modules/images/image-loading-optimization/storage/rest-api.php

```diff
@@ -130,24 +130,10 @@ function ilo_register_endpoint() {
 							'isLCPCandidate'     => array(
 								'type' => 'bool',
 							),
-							'breadcrumbs'        => array(
-								'type'     => 'array',
+							'xpath'              => array(
+								'type'     => 'string',
 								'required' => true,
-								'items'    => array(
-									'type'       => 'object',
-									'properties' => array(
-										'tag'   => array(
-											'type'     => 'string',
-											'required' => true,
-											'pattern'  => '^[a-zA-Z0-9-]+$',
-										),
-										'index' => array(
-											'type'     => 'int',
-											'required' => true,
-											'minimum'  => 0,
-										),
-									),
-								),
+								'pattern'  => '^(/\*\[\d+\]\[self::.+?\])+$', // See ILO_HTML_Tag_Processor::get_xpath() for format.
```

Comment:
Just a suggestion: Make this a `const` on `ILO_HTML_Tag_Processor` since that's where the definition really comes from.

Reply:
Good idea. Done in 2df915d8